### ISSUE TITLE
Music page shows incorrect data

### ISSUE THREAD

**Issue by zertap** [weirdgloop/WikiSync#7]
Examples:
I don't have dogfight unlocked, wiki page shows it as unlocked
I do have Dunes of Eternity unlocked, wiki page shows it is not unlocked

In addition, in-game I have 673/714 tracks unlocked and wiki page says 651/714 unlocked.

My rsn is zertap, feel free to use my data to debug the problem.

**Comment by andmcadams** [comment]
I think some of the music ids got shuffled around with a recent update. I can look into this sometime this week. I thought we had fixed them already but I guess not. Thanks for reporting :+1:

**Comment by andmcadams** [comment]
Alright, I think we fixed this. It was a combination of Jagex changing some of the scripts on their side and there being some incorrect ids set on the wiki page itself.

I checked your account @zertap and it seems like you now have 707/714 unlocked. Not sure if that's something messed up still on our end or if you were just going for the music cape and got a bunch of tracks in the last day or so. My accounts seem fine, but obviously thats not exhaustive. Let me know if you're still having issues, if not I'll close this.
